sql/src/SqlCsv.sk

```diff
@@ -624,12 +624,7 @@ fun applyDiffStrategy(
     isEligibleForTomb = (tick, source) ~>
       tick.value < snapshot.default(0) || source == writer;
 
-    !dir = dir.reset(
-      newRoot,
-      writer,
-      Array[].collect(SortedSet),
-      isEligibleForTomb,
-    );
+    !dir = dir.reset(newRoot, writer, SortedSet[], isEligibleForTomb);
```

Comment:
This works because `SortedSet[]` desugars to `SortedSet::createFromItems(Array[])`, right?

Reply:
Indeed! (source: `call_from_items` in `skipNaming.sk`)